ui/src/components/common/Holdings.tsx

```diff
@@ -199,6 +200,11 @@ type SplitFormProps = {
 const SplitForm: React.FC<SplitFormProps> = ({ deposit }) => {
     const { asset } = deposit.contractData;
     const ledger = useLedger();
+    const [ splitNumberError, setSplitNumberError ] = useState<string>()
+
+    const tokenQuantityPercision = Number(useStreamQuery(Token).contracts
```

Comment:
if the websocket fails here do we crash and burn? should we have a default precision or show some error?

Reply:
yep will add fo rthis